tests/draft4/dependencies.json

```diff
@@ -109,5 +109,43 @@
                 "valid": false
             }
         ]
+    },
+    {
```

Comment:
There was an additional test in the code in the appendix, which was not described in the paper, that tests what happens when dependences extend the schema with additonal rules. I think this isn't so much an edge case as something not covered by the test suite at present